Makefile

```diff
@@ -233,6 +235,9 @@ SRCS:=$(wildcard $(SRC)/*.cpp)
 SRCS_HMAC:=$(wildcard $(HMAC_SRC)/*.cpp)
 SRCS_CLKMGR:=$(wildcard $(CLKMGR_DIR)/[cupi]*/*.[ch]* $(CLKMGR_DIR)/*/*/*.h)
 COMP_DEPS:=$(OBJ_DIR) $(HEADERS_GEN_COMP)
+CLKMGR_LIB_NAME:=libclkmgr
+CLKMGR_BTH:=client_state clockmanager_c event_state subscription utility
```

Comment:
what is _BTH ?

Reply:
should not list down all the file manually, use something like CLKMGR_HEADERS := $(wildcard $(CLKMGR_PUB_x)/*)